.github/workflows/performance.yml

```diff
@@ -70,7 +70,7 @@ jobs:
                   WP_MAJOR="${WP_VERSION_ARRAY[0]}.${WP_VERSION_ARRAY[1]}"
                   ./bin/plugin/cli.js perf $GITHUB_SHA debd225d007f4e441ceec80fbd6fa96653f94737 --tests-branch $GITHUB_SHA --wp-version "$WP_MAJOR"
 
-            - uses: actions/github-script@d556feaca394842dc55e4734bf3bb9f685482fa0 # v6.3.3
+            - uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975 # v6.4.0
```

Comment:
Why is this change necessary in this PR?

Reply:
I don't get it, it was a [commit](https://github.com/WordPress/gutenberg/pull/47744/commits/22834076b43884f8b0eff2ea3998fa73191e5c5a) by the dependency bot??

Reply:
it disappeared with a rebase